ios/Lunes.xcodeproj/project.pbxproj

```diff
@@ -1048,6 +1055,7 @@
 					"\"$(inherited)\"",
 				);
 				MTL_ENABLE_DEBUG_INFO = NO;
+				PROVISIONING_PROFILE_SPECIFIER = None;
```

Comment:
Is this correct? Seems weird for release builds

Reply:
Release build is the next issue. Will be adjusted there, if necesarry.